api/controllers/userController.js

```diff
@@ -0,0 +1,21 @@
+const { userService } = require("../services/");
+const { catchAsync } = require("../utils/error");
+
+const signIn = catchAsync(async (req, res) => {
+  const { email, password } = req.body;
+
+  if (!email || !password) {
+    const error = new Error("KEY_ERROR");
+    error.status = 400;
+
+    throw error;
+  }
+
+  const accessToken = await userService.signIn(email, password);
+
+  return res.status(200).json({ accessToken: accessToken });
```

Comment:
```suggestion
  return res.status(200).json({ accessToken });
```
javascript에서는 객체의 property name과 value의 변수명이 같다면 위와 같이 간단하게 작성할 수도 있습니다.